std/string.d

```diff
@@ -2267,7 +2239,7 @@ unittest
         assert(s2 !is s1);
         s1 = to!S("\u0131 \u0130");
         s2 = capitalize(s1);
-        assert(cmp(s2, "\u0049 \u0069") == 0);
+        assert(cmp(s2, "\u0049 i\u0307") == 0);
```

Comment:
Honestly it was an assumption, coupled with the fact that it's what `asCapitalized ` outputs. It really doesn't make sense to me to think that `İ`'s lower case version is just a regular lower case i.